src/contents/shared/MyProfile.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import { ProfileEdit } from './ProfileEdit';
+
+class MyProfile extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isEdit: false
+        };
+        this.onEdit = this.onEdit.bind(this);
```

Comment:
Please use arrow functions instead of bind e.g.
`onEdit => () => { ... }`
If you write it like this, you don't need to bind functions anymore.

Reply:
Done!